modules/deploy-app.adoc

```diff
@@ -20,15 +20,15 @@ From the {product-title} web console, you can deploy a test application from the
 ifndef::quickstart[]
 .Prerequisites
 
-* You logged in to {cluster-manager-url}.
-* You created an {product-title} cluster.
+* You logged in to the {hybrid-console-url}.
+* You created a {product-title} cluster.
 * You configured an identity provider for your cluster.
 * You added your user account to the configured identity provider.
 endif::[]
 
 .Procedure
 
-. From the {cluster-manager} {hybrid-console-second}, click *Open console*.
+. From the *Clusters* page of the {hybrid-console-second}, click *Open console*.
```

Comment:
In Clusters page there is no option called "Open console". "Open console" will be  shown as context menu of a selected cluster. Refer inline screenshot
![image](https://github.com/openshift/openshift-docs/assets/93653094/db875ba0-a993-4185-aa7b-eb668e7c9b03)
I would request to update the step definition as per above mentioned flow.

Reply:
Thanks @jmekkatt, I update that.